### PR TITLE
VR-4592 dataset blobs description metadata proto

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

**Review by ravishetye — CHANGES_REQUESTED** [review]
this will change based on the implementation PR, we could use the VersioningCompositeIdentifier to attach label or description to it. So we need to add 3 more APIs for add,get,delete description

**Comment by evgeniilezhnin** [comment]
> this will change based on the implementation PR, we could use the VersioningCompositeIdentifier to attach label or description to it. So we need to add 3 more APIs for add,get,delete description

created separate endpoints for metadata parameters instead of labels (this will be used only for description parameter for now)

File: protos/protos/public/modeldb/metadata/MetadataService.proto [hunk 1]
```diff
@@ -51,6 +58,35 @@ message DeleteLabelsRequest {
     }
 }
 
+message GetParameterRequest {
```
Comment (ravishetye) [inline protos/protos/public/modeldb/metadata/MetadataService.proto]: replace Parameter with Metadata and then this looks good to me.

Comment (ravishetye) [reply protos/protos/public/modeldb/metadata/MetadataService.proto]: or `Property` to avoid the repetition of metadata/metadata

Comment (ravishetye) [reply protos/protos/public/modeldb/metadata/MetadataService.proto]: feel free to merge this if you end up making the change to Property

**Pull request merged**